test/parallel/test-fs-stream-destroy-emit-error.js

```diff
@@ -0,0 +1,41 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
IIRC all of this text shouldn't be added for new tests.

Reply:
Yeah, new source files should never include the copyright boilerplate

Reply:
I removed the copyright boilerplate. thanks!